components/feature/addons/src/main/java/mozilla/components/feature/addons/update/AddonUpdater.kt

```diff
@@ -413,6 +422,16 @@ class DefaultAddonUpdater(
                 .apply()
         }
 
+        @Synchronized
+        fun remove(context: Context, addonId: String) {
```

Comment:
nit: Let's make it consistent with method name here - `unmarkAsAllowed()`?

Reply:
👍  let's use `unmarkAsAllowed`